packages/ui/src/components/menu/menu.vue

```diff
@@ -52,43 +49,39 @@
       behavior: 'smooth',
     });
   }
-
-  const menuClasses = computed(() => {
-    return {
-      [`${props.type}`]: props.type,
-      [`is-${props.background}`]: props.background,
-      [`is-borderBottom`]: props.hideBottomBorder || props.hideBorders,
-      [`hide-borders`]: props.hideBorders,
-      collapsed: props.collapse,
-    };
-  });
 </script>
 
 <template>
   <nav
     ref="menuRef"
     class="acv-menu"
-    :class="menuClasses"
+    :class="{
+      [`acv-menu_type_${props.type}`]: true,
+      [`acv-menu_background_${props.background}`]: background,
```

Comment:
Because when I had it as other props sometimes we had "undefined", so I assume that we don't always have background set and better to check that and apply only when needed. Maybe I can change it to props.background to be more clear?

Reply:
ok, please change it